libraries/AC_Avoidance/AC_Avoidance_config.h

```diff
@@ -4,7 +4,7 @@
 #include <AC_Fence/AC_Fence_config.h>
 
 #ifndef AP_AVOIDANCE_ENABLED
-#define AP_AVOIDANCE_ENABLED AP_FENCE_ENABLED
```

Comment:
Maybe we should change this so it's (AP_FENCE_ENABLE || HAL_PROXIMITY_ENABLED).  No point in having avoidance if neither of these is available.

Reply:
I can see if this works - but this sort of thing usually ends up with include loops / tears.